AmazonEmailServiceRegions.pas

```diff
@@ -2,32 +2,32 @@
 
 interface
 
-uses
-  AmazonEmailService;
-
 type
   TAmazonEmailServiceRegions = class
   public
-    class function GetServiceURL(const Region: AwsRegions): string;
+    class function DefineServiceURL(const AWSRegionEndpoint: string): string;
```

Comment:
This method name is a bit misleading, I think it should be called `FormatServiceURL' because it is what this method does.